README.md

```diff
@@ -1,4 +1,4 @@
-TangeloHub [![Build Status](https://travis-ci.org/Kitware/tangelohub.png?branch=master)](https://travis-ci.org/Kitware/tangelohub)
+TangeloHub [![Build Status](https://travis-ci.org/Kitware/flow.png?branch=master)](https://travis-ci.org/Kitware/flow)
```

Comment:
Ok one more - how about changing the *name* TangeloHub (col 1 on this line) to Flow :)